src/components/cc-zone-picker/cc-zone-picker.js

```diff
@@ -0,0 +1,174 @@
+import { css, html } from 'lit';
+import { CcFormControlElement } from '../../lib/form/cc-form-control-element.abstract.js';
+import { accessibilityStyles } from '../../styles/accessibility.js';
+import { i18n } from '../../translations/translation.js';
+import '../cc-badge/cc-badge.js';
+import '../cc-icon/cc-icon.js';
+import '../cc-img/cc-img.js';
+import '../cc-notice/cc-notice.js';
+import '../cc-zone-card/cc-zone-card.js';
+
+import { iconRemixEarthLine as zoneIcon } from '../../assets/cc-remix.icons.js';
+import { isStringEmpty } from '../../lib/utils.js';
+
+/**
+ * @typedef {import('./cc-zone-picker.types.js').ZoneItem} ZoneItem
+ * @typedef {import('./cc-zone-picker.types.js').ZoneSection} ZoneSection
+ * @typedef {import('../../lib/events.types.js').EventWithTarget<HTMLInputElement>} HTMLInputElementEvent
+ */
+
+/**
+ * A component that allows you to select a zone from a list of zones sections.
+ *
+ * @cssdisplay block
+ */
+export class CcZonePicker extends CcFormControlElement {
+  static get properties() {
+    return {
+      ...super.properties,
+      value: { type: String },
+      zonesSection: { type: Array },
```

Comment:
**nitpick:**
```suggestion
      zoneSections: { type: Array },
```
or 
```suggestion
      zonesSections: { type: Array },
```

Reply:
I had a similar thought. We need plural on "section". Not sure about "zone". Both @florian-sanders-cc suggestions are OK.